js/wizard/view.js

```diff
@@ -304,8 +304,7 @@ OCA = OCA || {};
 		enableTabs: function() {
 			//do not use this function directly, use basicStatusCheck instead.
 			if(this.saveProcesses === 0) {
-				$('.ldap_action_continue').removeAttr('disabled');
-				$('.ldap_action_back').removeAttr('disabled');
+				$('.ldap_action_continue, .ldap_action_back').removeAttr('disabled');
```

Comment:
`.prop('disabled', false)` ? (a long time ago I was told to use `.prop` because it is more cross-browser compatible, this was in the dark times of IE8)